cmd/devnet/services/polygon/heimdallsim/heimdall_simulator_test.go

```diff
@@ -94,14 +94,17 @@ func TestSimulatorEvents(t *testing.T) {
 		t.Skip("fix me on win")
 	}
 
+	// the number of events included in v1-000000-000500-borevents.seg
+	eventsCount := 23
```

Comment:
@mh0lt  do you think it's right change?

Reply:
This number looks reasonable for the start of the chain which this test is referencing.  Sync events it seems where not very popular at the start in bor.